plugins/source/awspricing/resources/services.go

```diff
@@ -0,0 +1,320 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/cloudquery/cloudquery/plugins/source/awspricing/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+const ROOT_URL = "https://pricing.us-east-1.amazonaws.com"
+
+func Services() *schema.Table {
+	return &schema.Table{
+		Name:                "awspricing_services",
+		Resolver:            fetchServicesTable,
+		PreResourceResolver: getPricingFile,
+		Transform:           transformers.TransformWithStruct(&PricingFile{}, transformers.WithSkipFields("Products", "Terms"), transformers.WithPrimaryKeys("OfferCode", "Version", "PublicationDate")),
```

Comment:
```suggestion
		Transform: transformers.TransformWithStruct(&PricingFile{},
			transformers.WithSkipFields("Products", "Terms"),
			transformers.WithPrimaryKeys("OfferCode", "Version", "PublicationDate"),
		),
```
for the sake of formatting, non-blocking